osu.Server.Queues.ScorePump/Performance/Values/UpdateValuesScoresCommand.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Threading;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using McMaster.Extensions.CommandLineUtils;
+
+namespace osu.Server.Queues.ScorePump.Performance.Values
+{
+    [Command("scores", Description = "Computes pp of specific scores.")]
+    public class UpdateValuesScoresCommand : PerformanceCommand
+    {
+        [UsedImplicitly]
+        [Required]
+        [Argument(0, Description = "A space-separated list of scores to compute PP for.")]
```

Comment:
Is this actually still space separated? I think McMaster is comma separated (which I'd prefer because this screws me over every time i run it on the old processor)

Reply:
Yeah it's still space separated. I'll use a string and do manual parsing.